### PR TITLE
Add note about changes in check-in client

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

Describe what needs to be done for changing  configuration of clients in production

### CODE REVIEW

**Comment by enolfc** [comment]
@Pansanel please take a look

File: source/openstack.rst [hunk 1]
```diff
@@ -499,6 +499,14 @@ Once tests in the development instance of Check-in are successful, you can move
 
 * Update your client secret and client ID in the Apache configuration
 
+.. note::
+
+   **Changes in the production settings**
+
+   If you want to make any changes to the client configuration of the
+   production instance, first make the changes in the Check-in development
+   environment and then open a open a `GGUS ticket <https://ggus.eu>`_
```
Comment (gwarf) [inline source/openstack.rst]:
```suggestion
   environment and then open a `GGUS ticket <https://ggus.eu>`_
```

**Comment by enolfc** [comment]
how cool is that commit within the review! haven't seen it before

**Comment by enolfc** [comment]
@gwarf could you approve? Thanks!

**Pull request merged**